src/HonzaBotner.Database/Mappers/VerificationMapper.cs

```diff
@@ -9,5 +9,6 @@ public void Map(EntityTypeBuilder<Verification> builder)
         builder.HasKey(v => v.UserId);
         builder.HasIndex(v => v.AuthId)
             .IsUnique(); // By default it is "IS UNIQUE IF NOT NULL"
+        builder.Property(v => v.Username).IsRequired(false);
```

Comment:
Should also be `.IsUnique()`.

Reply:
This is very temporary, uniqueness is already checked by the hash.

Reply:
It would be easier to migrate with the constraint on username, but 🤷.